sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,16 +1,27 @@
 # Release History
 
-## 4.1.0-beta.2 (Unreleased)
+## 4.1.0-beta.2 (2023-05-10)
+### Bugs Fixed
+- Document Table cell now returns correct values for bounding boxes.
```

Comment:
which is already mentioned in the 4.0.7 (2023-05-10)

Reply:
yeah the fix will go in beta as well